ev3dev2/motor.py

```diff
@@ -1924,6 +2037,181 @@ def get_speed_steering(self, steering, speed):
         return (left_speed, right_speed)
 
 
+class MoveDifferential(MoveTank):
+    """
+    For all methods speed can be an integer or a SpeedInteger object
+    """
+
+    def __init__(self, left_motor_port, right_motor_port, wheel_class, wheel_distance_mm, desc=None, motor_class=LargeMotor):
+        MoveTank.__init__(self, left_motor_port, right_motor_port, desc, motor_class)
+        self.wheel = wheel_class()
+
+        # The distance between the mid point of the wheels.  You may need to
+        # do some test drives to find the correct value for your robot.  It
+        # is not as simple as measuring the distance between the midpoints of
+        # the two wheels. The weight of the robot, center of gravity, etc come
+        # into play.  You can use utils/move_differential.py to call on_arc_left()
+        # to do some test drives in 1-foot circles.  Tweak wheel_distam_mm until
```

Comment:
Using a 1-foot circle to find dimensions in millimeters? 🤔

Reply:
A 1-meter circle felt like overkill :) They can drive in whatever size circle they want, they just need to tweak the wheel_distance until it makes an exact circle.

Reply:
Ah, OK. That sounds good to me.